tests/test_dtcwt.py

```diff
@@ -115,12 +116,13 @@ def test_energy_preservation(alternate_gh):
     assert torch.abs(ratio - 1) <= 0.01
 
 
+@pytest.mark.parametrize("J", range(3))
 @pytest.mark.parametrize("alternate_gh", [True, False])
-def test_avrg_energy(alternate_gh):
+def test_avrg_energy(J, alternate_gh):
     '''
     Test the power of the signals for normalization case.
     '''
-    tfm = murenn.DTCWT(alternate_gh=alternate_gh, normalize=True)
+    tfm = murenn.DTCWT(J=J+1, alternate_gh=alternate_gh, normalize=True)
```

Comment:
i'd prefer if you had `J=J` in the callsite and `range(1, 4)` in the `pytest .mark` parametrization macro